src/main/java/com/untamedears/JukeAlert/command/commands/JaToggleLeversCommand.java

```diff
@@ -60,12 +57,9 @@ public boolean execute(CommandSender sender, String[] args) {
                     sender.sendMessage(ChatColor.RED + "Toggle Lever Settings can only be applied to logging jukeboxes.");
                     return false;
             	}
-            	
+            	JukeAlert plugin = JukeAlert.getInstance();
             	plugin.getJaLogger().updateSnitchToggleLevers(snitch, flag);
-            	Snitch newSnitch = snitch;
-            	newSnitch.setShouldToggleLevers(flag);
-            	plugin.getSnitchManager().removeSnitch(snitch);
-            	plugin.getSnitchManager().addSnitch(newSnitch);
+            	snitch.setShouldToggleLevers(flag);
```

Comment:
I wonder if there's a reason for that prior pattern? no? Perhaps its immutable once created?